src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -80,34 +80,34 @@ internal static void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture, E
             if (errorLogger != null)
             {
 #pragma warning disable RS0013 // We need to invoke Diagnostic.Descriptor here to log all the metadata properties of the diagnostic.
-                var issue = new Issue(diagnostic.Id, diagnostic.GetMessage(culture),
+                var result = new Result(diagnostic.Id, diagnostic.GetMessage(culture),
```

Comment:
It'd be helpful if we could persist the Roslyn diagnostic format specifier and individual message arguments, to save space by supporting the SARIF formattedMessage mechanism. we'd need a change to roslyn for this, obvious, might be worth opening a tracking item for the possibility.